outputs/discord.go

```diff
@@ -0,0 +1,30 @@
+package outputs
+
+import (
+	"github.com/falcosecurity/falcosidekick/types"
+)
+
+type discordPayload struct {
+	Content    string `json:"content"`
+	Avatar_url string `json:"avatar_url,omitempty"`
+}
+
```

Comment:
Is it possible to enhance that payload to get a formatted message with a color according to priority and embeds?

See : https://discord.com/developers/docs/resources/webhook

Reply:
I'll probably add *some* of the configurability of the slack hook but I don't want to let "perfect be the enemy of the good" if that makes sense.

Reply:
I agree with you, but it's important to me to have nice output, it was my main purpose at start.

Reply:
So it doesn't look like discord has the same priority hooks that Slack has. There are some hacks (and I can't emphasize enough that they hare hacks) at [this link](https://www.writebots.com/discord-text-formatting/). Red seems easy and not-terrible if you'd like me to add that.

Reply:
I thought it was possible with embed objects : https://discord.com/developers/docs/resources/channel#embed-object

Here a result : https://birdie0.github.io/discord-webhooks-guide/discord_webhook.html

Reply:
Oh Cool! Thanks for the example. I messed with `embed` for a minute but couldn't get it working. I'll try to incorporate.